test/integration/cypress/support/Path.ts

```diff
@@ -0,0 +1,5 @@
+export type KgPath = {
```

Comment:
File name needs to match type.

Reply:
If you wouldn't mind changing Node and Edge as well, that would be much appreciated.  I made the same mistake there.

Reply:
Gotcha will do